src/main/java/duke/DukeTest.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import org.junit.jupiter.api.Assertions;
+import org.testng.annotations.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+    @Test
+    public void dummyTest() {
+        Assertions.assertEquals(2, 2);
+    }
+
+    @Test
+    public void toDoTest() {
```

Comment:
Minor edit: Maybe you could consider renaming your testcases in the format just to make it more clear:
featureUnderTest_testScenario_expectedBehavior()
Source: https://se-education.org/guides/conventions/java/intermediate.html#naming